inference/README.md

```diff
@@ -26,31 +26,37 @@ docker compose --profile inference up -d
 Tail the logs:
 
 ```shell
+# cd /root/of/this/repository
 docker compose logs -f    \
     inference-server      \
     inference-worker      \
-    inference-text-client \
-    inference-text-generation-server
+    inference-safety
 ```
 
-Attach to the text-client, and start chatting:
-
-```shell
-docker attach open-assistant-inference-text-client-1
-```
-
-> **Note:** In the last step, `open-assistant-inference-text-client-1` refers to
-> the name of the `text-client` container started in step 2.
-
 > **Note:** The compose file contains the bind mounts enabling you to develop on
 > the modules of the inference stack, and the `oasst-shared` package, without
 > rebuilding.
 
+> **Note:** You can change the model by editing variable `MODEL_CONFIG_NAME` in
+> the `docker-compose.yaml` file. Valid model names can be found at
+> https://huggingface.co/OpenAssistant .
+
```

Comment:
huggingface only gives the model names, the valid model config names mapping to the model names, can be found in https://github.com/LAION-AI/Open-Assistant/blob/main/oasst-shared/oasst_shared/model_configs.py#L23

Reply:
Will fix.